app/services/analytics_update_service.rb

```diff
@@ -13,8 +13,8 @@ def initialize
     @service.authorization = authorization
   end
 
-  def add_new_service(api_key, department, service, options = { spreadsheet_id: Rails.application.secrets.spreadsheet_id, range: Rails.application.secrets.spreadsheet_range })
-    department ||= '(not set)'
+  def add_new_service(api_key, department, service, is_government, options = { spreadsheet_id: Rails.application.secrets.spreadsheet_id, range: Rails.application.secrets.spreadsheet_range })
+    department ||= is_government == false ? 'Non government' : '(not set)'
```

Comment:
Does this conditional match with the change on this pr https://github.com/openregister/registers-frontend/pull/272 ?

Reply:
yes, please see comment: https://github.com/openregister/registers-frontend/pull/272#discussion_r184945280